app/components/goals-list/goals-list.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const goalsList = props => {
+
+  const goals = props.store.goals;
```

Comment:
So here the goal list component should be getting exacrly the props it needs 

```js
const goalsList = props => {

const goals = props.goals;
```
or
```js
const goalsList = props => {

const { goals }  = props.goals;
```

or
```js
const goalsList = ({ goals }) =>
```